packages/communication-react/CHANGELOG.stable.md

```diff
@@ -10,24 +10,20 @@ Mon, 13 Jan 2025 19:28:45 GMT
 [Compare changes](https://github.com/azure/communication-ui-library/compare/1.22.0...1.23.0)
 
 ### Features
-- We are excited to announce that the Azure Communication Services Web UI Library now supports the Media Access feature. This feature allows organizers, co-organizers, and presenters to control the ability of other attendees to send audio and video while in a call. Attendees can determine if their audio or video is enabled or disabled and check the media access status of other participants. Developers can integrate this functionality through our composites (e.g., CallComposite, CallWithChatComposite) as well as through components. ([PR #5546](https://github.com/azure/communication-ui-library/pull/5546) by fuyan@microsoft.com)
-- Hard mute feature UI changes ([PR #5452](https://github.com/azure/communication-ui-library/pull/5452) by fuyan@microsoft.com)
+- We are excited to announce that the Azure Communication Services Web UI Library now supports the Media Access feature. This feature allows organizers, co-organizers, and presenters to control the ability of other attendees to send audio and video while in a call. Attendees can determine if their audio or video is enabled or disabled and check the media access status of other participants. Developers can integrate this functionality through our composites (e.g., CallComposite, CallWithChatComposite) as well as through components. ([PR #5546](https://github.com/azure/communication-ui-library/pull/5546) by fuyan@microsoft.com, [PR #5452](https://github.com/azure/communication-ui-library/pull/5452) by fuyan@microsoft.com, [PR #5531](https://github.com/azure/communication-ui-library/pull/5531) by fuyan@microsoft.com, [PR #5456](https://github.com/azure/communication-ui-library/pull/5456) by fuyan@microsoft.com, [PR #5452](https://github.com/azure/communication-ui-library/pull/5452) by fuyan@microsoft.com, [PR #5539](https://github.com/azure/communication-ui-library/pull/5539) by fuyan@microsoft.com, [PR #5543](https://github.com/azure/communication-ui-library/pull/5543))
```

Comment:
Do we need to include all PRs? If so, we probably can skip those not connected to the functionality (like tests, storybook)